src/fides/api/schemas/consentable_item.py

```diff
@@ -83,7 +83,9 @@ class ConsentWebhookResult(BaseModel):
     A wrapper class for the identity map and notice map values returned from a `PROCESS_CONSENT_WEBHOOK` function.
     """
 
-    identity_map: Dict[str, Any] = {}
+    identity_map: Dict[
+        Literal["email", "phone_number", "fides_user_device", "external_id"], str
+    ] = {}
```

Comment:
Open to suggestions here. This identity map is converted to kwargs for `save_privacy_preferences_from_endpoint` in Fidesplus https://github.com/ethyca/fidesplus/pull/1563/files#diff-b41570e64a5a6483a62b6defd81e4cedba6f8ad778923245deb2060b7de008d3R252. It just seems a bit fragile as it is and especially difficult to track across two repos.